### PR TITLE
Enable chaos testing on net-contour

### PR DESCRIPTION
**Pull request by mattmoor**
/hold

### CODE REVIEW

**Comment by mattmoor** [comment]
Passed, let's hit it again.

/test pull-knative-sandbox-net-contour-integration-tests

**Comment by mattmoor** [comment]
Passed, let's hit it again.

/test pull-knative-sandbox-net-contour-integration-tests

**Comment by mattmoor** [comment]
Passed, let's hit it again.

/test pull-knative-sandbox-net-contour-integration-tests

File: test/config/chaosduck.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- kind: ServiceAccount
+  name: chaosduck
+  namespace: knative-serving
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: chaosduck
+  namespace: knative-serving
```
Comment (n3wscott) [inline test/config/chaosduck.yaml:61]: just double checking... does net-contour install into `knative-serving`?

Comment (n3wscott) [reply test/config/chaosduck.yaml:61]: should we make a `chaos` namespace?

Comment (mattmoor) [reply test/config/chaosduck.yaml:61]: yes

Comment (mattmoor) [reply test/config/chaosduck.yaml:61]: no, we can install this into several namespaces where we want it to wreak havok 😉

**Pull request merged**